unit/constant/generate_constants.go

```diff
@@ -0,0 +1,327 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
```

Comment:
I think we should make these values accurate to the May 2019 redefinition (even if we're a bit early). We should think about what sort of compatibility guarantees we have. I would rather us update as changes happen / science gets better, but it's not clear what the right behavior is.

Also, did you have a standard practice for defining the constants? Would be nice to cite a reference or some canonical source. This question is especially related to the "uncertain" digits, where dropping them is effectively the same as setting them to zero.

Reply:
I trawled WP. Is there a good standard document?

Reply:
NIST releases a guide: https://physics.nist.gov/cuu/index.html , but it's not up to date with the SI redefinition

Reply:
What would you prefer wrt uncertainty?

I'll add some notes to the doc.go talking about stability of values.

Reply:
I think for the purposes of compatibility it makes sense to say in the docs that we use the most recent NIST values (and not precede them). This gives us a clear reason for changes.